### ISSUE TITLE
close keyboard?

### ISSUE THREAD

**Issue by markshust** [mnill/cordova-custom-keyboard#1]
How does one close the keyboard after data input?

**Comment by mnill** [comment]
Click into webview?

**Comment by markshust** [comment]
I'm not sure I understand. We have no webview in safari ;)

**Comment by markshust** [comment]
I'm thinking there should be an equivalent `CustomKeyboard.close`?

**Comment by mnill** [comment]
I am sorry, it is plugin for cordova. Cordova - html5 crossplatform applications platform based on webviews. It will not working in safari.

**Comment by markshust** [comment]
I'm sorry -- I meant embedded browser in Simulator.

Once CustomKeyboard is triggered, it never closes, no matter what is focused on. I'm just showing the simulator persisting the keyboard no matter what I do:

![screen shot 2016-05-03 at 12 53 05 pm](https://cloud.githubusercontent.com/assets/437029/14991169/0932e35a-112e-11e6-9d71-5e3c1c9193a2.png)

I'd expect on blur, it to close.

**Comment by mnill** [comment]
On my devices all works fine(i am not tested on simulators, will try it later). Are you sure you correctly install and use my plugin? Decemical(not number keyboard) looks like this: 
![img_0020](https://cloud.githubusercontent.com/assets/3879603/14991274/9e10cce2-1171-11e6-8c3d-a114994b75b5.jpg)

With dot symbol in left bot, you have opened standard number keyboard.

**Comment by markshust** [comment]
Yes, pretty sure i'm triggering it correctly. In this case, I just wanted the keypad:

```
    if (_.isObject(CustomKeyboard)) {
      CustomKeyboard.open(this.state.numSingles, 5, function (value) {
        console.log('custom', value)
      });
    }
```

The keypad shows correctly (even all the different types), however it doesn't close automatically on blur.

**Comment by mnill** [comment]
It is strange, i tested just right now. All works fine, even in simulator.

Anyway i realized 'close' method for you.

If you want to contribute, or anyone also faced with same issue there is minimum worked project with pure cordova for testing.

[keyboard.zip](https://github.com/mnill/cordova-custom-keyboard/files/247393/keyboard.zip)

**Comment by rafaellop** [comment]
I also cannot close the keyboard. Both in the sim and on a device (the newest ios). What's interesting the keyboard closes when taping on some html elements for example title bar or another input which, but not on some other without a particular case.

**Comment by mnill** [comment]
Looks like invisible full screen div can resolve this issue.
